data-processing-lib/test/data_processing_tests/ray/launcher_test.py

```diff
@@ -44,7 +44,7 @@
 code_location = {"github": "github", "commit_hash": "12345", "path": "path"}
 
 
-class TestLauncher(TransformLauncher):
+class TestLauncher(TransformLauncherRay):
```

Comment:
This is a general common about all ray launcher esting.  our classes need generalization to not only use/create ray-based launchers and then add cases for the pure python launcher.

Reply:
added